assets/src/util/util.tsx

```diff
@@ -25,7 +25,10 @@ export const formatTimeString = (timeString: string) =>
   moment(timeString).tz("America/New_York").format("h:mm");
 
 export const imagePath = (fileName: string): string =>
-  isOFM() ? `images/${fileName}` : `/images/${fileName}`;
+  isOFM() ? outfrontImagePath(fileName) : `/images/${fileName}`;
```

Comment:
If we're still using `imagePath` for OFMs in addition to `import`ing images into components, in which scenarios do we use which one?

Reply:
The `import` should only have to be done once in `outfront_evergreen_content.tsx`. It gives webpack the ability to preload the images and save them to the [correct folder](https://github.com/mbta/screens/blob/a2839ceea385ea916fa470dfdfc528783dfcab4b/assets/webpack.config.js#L251). Because this folder only exists in the packaged app, we only need to reference that path on live triptychs. Eventually, DUPs should be treated similarly, but I wanted to leave them alone for now.